setup.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+
+"""
+Standard build script.
```

Comment:
hahaha oks, no problem :)

As a said above, to release a new version to Pypi, we'll need to do:

**Register the new package**
If we want to unify the python packages versions (look the image I put above) maybe we'll need to delete the rest of version from Pypi. You'll must login in at:

https://pypi.python.org/pypi?%3Aaction=login_form
And remove the old versions.

Then It'll necessary to register the new version, running (in python 2 o 3):
`python setup.py sdist register upload`

**Uploads**
In next releases, we only need to do 2 things:
- Change the version param in setup(..). If there's no change of this value, Pypi will reject the update, because this version is already available in Pypi.
- Run then command: `python setup.py sdist upload`